berserk/types/account.py

```diff
@@ -120,3 +121,14 @@ class UserPreferences(TypedDict, total=False):
 class Preferences(TypedDict):
     prefs: UserPreferences
     language: str
+
+
+class TimelineEntry(TypedDict):
+    type: NotRequired[str]
+    data: NotRequired[dict[str, str] | None]
+    date: int
```

Comment:
these are required by the docs https://lichess.org/api#tag/account/get/apitimeline